script.js

```diff
@@ -0,0 +1,166 @@
+            <hr class="breakthrough">
+            <p class="singers-achievement">${singer.achievement}</p>
+        </div>
+    </section>`;
+});
+
+singersCard.innerHTML = content1;
+
+const secondSingersArray = [
+  {
+    id: 0,
+    img: 'images/damien Escobar.jpg',
+    alt: 'Escobar',
+    name: 'Damien Escobar',
+    description: 'Damien Esco, is an American violinist',
+    achievement: 'Known for his crossover violin musical style consists of a mix of classical, jazz, pop, R&B, and hip hop',
+  },
+  {
+    id: 1,
+    img: 'images/Chrissette michelle.jpeg',
+    alt: 'Michele',
+    name: 'Chrisette Michele',
+    description: 'An American R&B and soul singer',
+    achievement: 'won a Grammy Award for Best Urban/Alternative Performance in 2009 for her song, "Be Ok"',
+  },
+];
+
+const singersCard2 = document.querySelector('.singers-card1');
+
+let content2 = '';
+
+secondSingersArray.forEach((singer) => {
+  content2 += `
+    <section class="singers">
+        <div class = "singers-container">
+            <div class="singers-front"></div>
+            <div class="singers-background">
+                <img src="${singer.img}" alt="${singer.alt}" class="image">
+            </div>
+        </div>
+        <div class="singers-info">
+            <h5 class="singers-name">${singer.name}</h5>
+            <p class="singers-description">${singer.description}</p>
+            <hr class="breakthrough">
+            <p class="singers-achievement">${singer.achievement}</p>
+        </div>
+    </section>`;
+});
+
+singersCard2.innerHTML = content2;
+
+const thirdSingersArray = [
+  {
+    id: 0,
+    img: 'images/eric benet.jpg',
+    alt: 'Eric',
+    name: 'Eric Benet',
+    description: ' American R&B/neo soul singer-songwriter and actor',
+    achievement: 'Benet has received a total of four Grammy nominations to date for his musical work.',
+  },
+  {
+    id: 1,
+    img: 'images/regina belle.png',
+    alt: 'Regina',
+    name: 'Regina Belle',
+    description: 'American singer-songwriter who started her career in the mid-1980s',
+    achievement: 'Known for her singles "Baby Come to Me" and "Make It Like It Was", Belle\'s most notable for two hit duets',
+  },
+];
+
+const sectionsCard3 = document.querySelector('.singers-card2');
+
+let content3 = '';
+
+thirdSingersArray.forEach((singer) => {
+  content3 += `
+      <section class="singers">
+          <div class = "singers-container">
+              <div class="singers-front"></div>
+              <div class="singers-background">
+                  <img src="${singer.img}" alt="${singer.alt}" class="image">
+              </div>
+          </div>
+          <div class="singers-info">
+              <h5 class="singers-name">${singer.name}</h5>
+              <p class="singers-description">${singer.description}</p>
+              <hr class="breakthrough">
+              <p class="singers-achievement">${singer.achievement}</p>
+          </div>
+      </section>`;
+});
```

Comment:
- [ ] Kindly make sure to not repeat yourself, `one array` for singers, and one `forEach` loop would create the singer section,